ui/tract-renderer/src/main/java/org/cru/godtools/tract/activity/TractActivity.java

```diff
@@ -860,7 +858,7 @@ private String getPagePathPartForSharing(@NonNull final Manifest manifest) {
     void trackTractPage(@NonNull final Page page, @Nullable final Card card) {
         final Manifest manifest = page.getManifest();
         mEventBus.post(new TractPageAnalyticsScreenEvent(manifest.getCode(), manifest.getLocale(), page.getPosition(),
-                                                         card != null ? card.getPosition() : null));
+                card != null ? card.getPosition() : null));
```

Comment:
another unrelated formatting change

Reply:
I use the Reformat Code to reformat all of the switch statements that I removed and it reformatted all of the extra changes that you see.

Reply:
there is a way to have reformat code only reformat lines that have been changed